tests/integration/targets/vmware_cluster_info/tasks/main.yml

```diff
@@ -0,0 +1,79 @@
+---
+- name: Test On Simulator
+  when: run_on_simulator
+  block:
+    - name: Create Cluster
+      vmware.vmware.cluster:
+        validate_certs: false
+        hostname: "{{ vcenter_hostname }}"
+        username: "{{ vcenter_username }}"
+        password: "{{ vcenter_password }}"
+        datacenter: "{{ vcenter_datacenter }}"
+        cluster: "{{ test_cluster }}"
+        port: "{{ vcenter_port }}"
+        state: present
+      register: _create
+    - name: Check Cluster Output
+      ansible.builtin.assert:
+        that:
+          - _create.cluster.moid is defined
+    # deleting a cluster in the simulator is not supporter
```

Comment:
Should it be add `Gather Cluster Info` task? Otherwise the test does not test the new module.